apps/extension/eslint.config.mjs

```diff
@@ -1,13 +1,18 @@
 import { penumbraEslintConfig } from '@penumbra-zone/eslint-config';
 import { config, parser } from 'typescript-eslint';
 
-export default config({
-  ...penumbraEslintConfig,
-  languageOptions: {
-    parser,
-    parserOptions: {
-      project: true,
-      tsconfigRootDir: import.meta.dirname,
+export default config(
+  {
+    ...penumbraEslintConfig,
+    languageOptions: {
+      parser,
+      parserOptions: {
+        project: true,
+        tsconfigRootDir: import.meta.dirname,
+      },
     },
   },
-});
+  {
+    ignores: ['**/message-external.ts'],
+  },
```

Comment:
Alternatively, think you could add `/* eslint-disable */` to the top of the file.